src/test/resources/features/users/create_users.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Create Users  
+
+  In order to create an account
+  As a valid Converter customer
+
+  @functional
```

Comment:
Please consider adding an additional tag that will indicate a postcondition to delete the user created in the test, for example: `@deleteUser`

Reply:
Thanks, It's done.